holoviews/core/dimension.py

```diff
@@ -783,35 +783,20 @@ class to be associated with dimensions. The contents associated
     _dim_aliases = dict(key_dimensions='kdims', value_dimensions='vdims',
                         constant_dimensions='cdims', deep_dimensions='ddims')
 
-
-    # Long-name aliases
-
-    @property
-    def key_dimensions(self): return self.kdims
-
-    @property
-    def value_dimensions(self): return self.vdims
-
-    @property
-    def constant_dimensions(self): return self.cdims
-
-    @property
-    def deep_dimensions(self): return self.ddims
-
```

Comment:
I suppose we haven't used them in a long time so they should be ok to remove now.

Reply:
You would think that, turns out Columnar_Data still had a reference:

      3 error_hmap = hv.HoloMap({(i, j): hv.Image(j*np.random.randn(3, 3), bounds=extents)
      4                          for i, j in product(range(3), np.linspace(0, 1, 3))},
      5                         key_dimensions=['Observation', 'noise'])